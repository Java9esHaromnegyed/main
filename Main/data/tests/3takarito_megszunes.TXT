loadArena testArena1.txt
initArena
addCleanerRobot 1 1 2
tick 1