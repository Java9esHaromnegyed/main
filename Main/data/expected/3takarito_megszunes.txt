arenaLoaded src: testArena3.txt
robotAdded id: 0 name: "player0" pos: [1; 0]
arenaInited
cleanerRobotAdded id: 1 pos: [1; 2]
tick
robotMoved id: 0 pos: [1; 1]
tick
robotMoved id: 0 pos: [1; 2]
cleanerRobotDied id: 1
obstacleAdded type: OIL  pos: [1; 2]
robotSlipped id: 0
oilAged: 1 pos: [1; 2]
